apps/demo/components/preview-content.tsx

```diff
@@ -38,7 +39,7 @@ export function PreviewContent() {
       <div className="w-full max-w-[400px] py-8 mx-auto flex-grow flex-1 flex items-center justify-center">
         {shouldRender ? (
           <>
-            {/* @ts-ignore */}
+            {/* @ts-expect-error - Property 'w3m-modal' does not exist on type 'JSX.IntrinsicElements' */}
```

Comment:
This is weird, shouldn't it be auto adding itself to the declared types alreadt?

Reply:
We have types for exported components, but w3m-modal is not one of them, its internal component, but maybe I could do export for this too yeah